automatic/gitextensions/update.ps1

```diff
@@ -19,27 +20,37 @@ function global:au_SearchReplace {
 }
 
 function global:au_GetLatest {
-  $download_page = Invoke-WebRequest -Uri 'https://github.com/gitextensions/gitextensions/releases' -UseBasicParsing
+  $LatestRelease = Get-GitHubRelease -Owner "gitextensions" -Name "gitextensions"
 
-  $urls = $download_page.Links | ? href -match 'GitExtensions-(.+)\.msi$' | select -expand href | % { 'https://github.com' + $_ }
+  $re = 'GitExtensions-(.+)\.msi$'
+  $downloadUrl = $LatestRelease.assets.browser_download_url | Where-Object { $_ -match $re } | select -First 1
+  $releaseUrl = $LatestRelease.html_url
+  
+  $version = ($downloadUrl -split '\/' | select -last 1 -skip 1).Substring(1)
 
-  $streams = @{}
-  $urls | % {
-    $version = $_ -split '\/' | select -last 1 -skip 1
-    if ($version -match '\.[a-z][a-z\d]*$') {
-      $version = $version -replace '\.([a-z][a-z\d]*)$', "-`$1"
-    }
-    $version = Get-Version $version
+  $version = $version -replace ".RC", "-RC"
 
-    if (!($streams.ContainsKey($version.ToString(2)))) {
-      $streams.Add($version.ToString(2), @{
-          Version = $version.ToString()
-          URL32   = $_
-        })
-    }
+  if(($version.ToCharArray() | Where-Object {$_ -eq '.'} | Measure-Object).Count -eq 0) {
+    $version = $version + ".0.0"
+  } elseif(($version.ToCharArray() | Where-Object {$_ -eq '.'} | Measure-Object).Count -eq 1) {
+    $version = $version + ".0"
+  }
+
+  $pre = ""
+  if ($version -match '^.*(-.*?)\..*$') {
+    $pre = $version -replace '^.*(-.*?)\..*$', "`$1"
+    $version = $version -replace $pre, ""
   }
 
-  return @{ Streams = $streams }
+  $version = $version + $pre
+  
+  $version = Get-Version $version
+
+  return @{
+    Version        = $version
+    URL32          = $downloadUrl 
+    ReleaseURL     = $releaseUrl
+  }
```

Comment:
Purely a note, for now, this removes the support for streams which we probably would want to add back at some point in the future.

@BlythMeister Can you please open a new issue about that so it can be looked into supporting streams? (it would probably need a new script helper to achieve that).

Reply:
I can't see how to create issue, only discussions.

Reply:
Sorry, I should have been clearer. It is a discussion that is expected to be created.
If you can create one of them, I can then create an issue based on that discussion.